app/controllers/direct_messages_controller.rb

```diff
@@ -31,9 +31,16 @@ def index
   def users_from_usernames(current_user, params)
     params.require(:usernames)
 
+    usernames = if params[:usernames].is_a?(String)
+      params[:usernames].split(",")
+    else
+      params[:usernames]
+    end
```

Comment:
😢

Reply:
Yeah not great but both actions are using this so not a lot we can do really.